seacrowd/sea_datasets/vihealthqa/vihealthqa.py

```diff
@@ -0,0 +1,153 @@
+# coding=utf-8
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@InProceedings{nguyen2022viheathqa,
+  author="Nguyen, Nhung Thi-Hong
+  and Ha, Phuong Phan-Dieu
+  and Nguyen, Luan Thanh
+  and Van Nguyen, Kiet
+  and Nguyen, Ngan Luu-Thuy",
+  title="SPBERTQA: A Two-Stage Question Answering System Based on Sentence Transformers for Medical Texts",
+  booktitle="Knowledge Science, Engineering and Management",
+  year="2022",
+  publisher="Springer International Publishing",
+  address="Cham",
+  pages="371--382",
+  isbn="978-3-031-10986-7"
+}
+"""
+_DATASETNAME = "vihealthqa"
+_DESCRIPTION = """\
+Vietnamese Visual Question Answering (ViVQA) consist of 10328 images and 15000 question-answer
+pairs in Vietnamese for evaluating Vietnamese VQA models. This dataset is built based on 10328 randomly
+selected images from MS COCO dataset. The question-answer pairs were based on the COCO-QA dataset that
+was automatically translated from English to Vietnamese.
+"""
+_HOMEPAGE = "https://huggingface.co/datasets/tarudesu/ViHealthQA"
+_LANGUAGES = ["vie"]
+_LICENSE = Licenses.UNKNOWN.value
+_PAPER_URL = "https://link.springer.com/chapter/10.1007/978-3-031-10986-7_30"
+_LOCAL = False
+_URLS = {
+    "vihealthqa": {
+        "train": "https://huggingface.co/datasets/tarudesu/ViHealthQA/raw/main/train.csv",
+        "val": "https://huggingface.co/datasets/tarudesu/ViHealthQA/raw/main/val.csv",
+        "test": "https://huggingface.co/datasets/tarudesu/ViHealthQA/raw/main/test.csv",
+    }
+}
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ViHealthQADataset(datasets.GeneratorBasedBuilder):
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
```

Comment:
Please add class docstring containing short description of the dataset.

Reply:
Sorry, take a long vocation in China, so not conenient to connect to the github. Updated